drivers/dose/dose.c

```diff
@@ -106,11 +106,100 @@ static inline void _disable_sense(dose_t *ctx)
 #endif
 }
 
+
+#ifdef MODULE_DOSE_WATCHDOG
+static unsigned _watchdog_users;
+static dose_t *_dose_base;
+static uint8_t _dose_numof;
+
+#ifndef DOSE_TIMER_DEV
+static xtimer_t _wdt;
+static unsigned _wdt_period;
+#endif
+
+static inline void _watchdog_start(void)
+{
+    if (_watchdog_users) {
+        return;
+    }
+
+    ++_watchdog_users;
```

Comment:
```suggestion
    _watchdog_users++;
```

Reply:
Feel free to ignore, I just find it weird to use when not needed, but just personal nitpicks.